.travis.yml

```diff
@@ -3,8 +3,13 @@ language: swift
 os:
   - osx
 
-osx_image: xcode10.3
-env: IPHONE_SIMULATOR_SDK='iphonesimulator12.4' OS_VERSION='12.4' IPHONE_NAME='iPhone Xs'
+matrix:
+  include:
+    - osx_image: xcode10.3
+      env: IPHONE_SIMULATOR_SDK='iphonesimulator12.4' OS_VERSION='12.4' IPHONE_NAME='iPhone Xs'
+    - osx_image: xcode11.5
+      env: IPHONE_SIMULATOR_SDK='iphonesimulator13.5' OS_VERSION='13.5' IPHONE_NAME='iPhone 11'
```

Comment:
Do we want to an iPhone 11 Pro which matches the same resolution of an iPhone Xs instead of iPhone 11 Pro?

Reply:
Done